### PR TITLE
Fix StudentId error message

### LINKED ISSUE #305: [PE-D] Adding a user with a non-standard student ID gives an error that has not enough information
Screenshot 2021-10-29 at 3.43.18 PM

In the screenshot, the student id does have 9 alphanumeric characters but the error says that it doesn't. 

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: wpinrui/ped#2

### PR DESCRIPTION
**Pull request by erwinqxy**
Fix #305
Screenshot 2021-10-30 at 12 23 20 PM

### CODE REVIEW

**Review by reddevil1313 — APPROVED** [review]
LGTM

**Comment by reddevil1313** [comment]
Pull latest version and solve merge conflicts @erwinqxy

File: src/test/java/seedu/programmer/model/FileManagerTest.java [hunk 1]
```diff
@@ -63,7 +63,10 @@ public void getStudentsFromCsv_wrongNumberOfColumns_throwsIllegalValueException(
 
     @Test
     public void getStudentsFromCsv_invalidField_throwsIllegalArgumentException() {
-        String expectedMessage = "Student ID should only contain 9 alphanumeric characters, and it should not be blank";
+        String expectedMessage = "Student ID should only contain 9 alphanumeric characters, "
+                                + "where the first and last characters "
+                                + " should be alphabets and the rest are numbers. And it should not be blank."
+                                + "\nFor example: A0212345T";
```
Comment (AllardQuek) [inline src/test/java/seedu/programmer/model/FileManagerTest.java:69]: How about "It should be of the form `AXXXXXXXY` where X is a non-negative integer and Y is the last character of your student ID." 

?

Comment (erwinqxy) [reply src/test/java/seedu/programmer/model/FileManagerTest.java:69]: sounds good! i'll do that

**Review by reddevil1313 — APPROVED** [review]
LGTM

**Pull request merged**